app/src/main/java/com/duckduckgo/app/location/ui/SiteLocationPermissionDialog.kt

```diff
@@ -74,8 +73,9 @@ class SiteLocationPermissionDialog : DialogFragment() {
 
     override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
         val binding = ContentSiteLocationPermissionDialogBinding.inflate(layoutInflater, null, false)
-
-        val alertDialog = AlertDialog.Builder(requireActivity()).setView(binding.root)
+        val alertDialog = MaterialAlertDialogBuilder(requireActivity(), com.duckduckgo.mobile.android.R.style.Widget_DuckDuckGo_Dialog).setView(
```

Comment:
Shouldn't be this a `CustomAlertDialogBuilder`?

Reply:
I want to leave it as is before we refactor anything. This dialog is bound to change and I want to keep as simple as possible